solidity/contracts/isms/multisig/AbstractMultisigIsm.sol

```diff
@@ -122,4 +82,41 @@ abstract contract AbstractMultisigIsm is IMultisigIsm {
         }
         return true;
     }
+
+    function computeDigest(bytes calldata _metadata, bytes calldata _message)
+        internal
+        pure
+        returns (bytes32)
+    {
+        // signed checkpoint properties
+        bytes32 root;
+        uint32 index;
+        bytes32 messageId;
+
+        if (
+            MultisigIsmMetadata.suffixType(_metadata) ==
+            MultisigIsmMetadata.SuffixType.ROOT
+        ) {
+            root = MultisigIsmMetadata.root(_metadata);
```

Comment:
Please add a comment explaining why things are where they are for this verification type (and below for the other)